ucm2/Intel/sof-hda-dsp/dsp.conf

```diff
@@ -32,13 +32,23 @@ If.SOFIPCVer {
 	      PostMixerAnalogPlaybackFIRBytes "Post Mixer Analog Playback FIR Eq bytes"
 	      PostMixerAnalogPlaybackDRCBytes "Post Mixer Analog Playback DRC bytes"
 	      PostMixerAnalogPlaybackDRCSwitch "Post Mixer Analog Playback DRC switch"
+	      Dmic0CaptureDRCBytes "Dmic0 Capture DRC bytes"
```

Comment:
Better to change this. The SDW PCs can also have PCH-DMIC and their topologies do no have same "Post Mixer Analog" class controls. So better to handle DMIC processing separately for code reuse.